modules/cnf-du-configuring-workload-partitioning.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-configuring-workload-partitioning_{context}"]
+
+= Configuring workload partitioning
+
+The following procedure outlines a high level, end to end workflow that deploys a cluster with workload partitioning enabled and pods that are correctly scheduled to run on the management CPU partition.
+
+. The user creates a machine config manifest to configure CRI-O to partition management workloads. The cpuset specified must match the reserved cpuset specified in the performance-addon-operator profile.
```

Comment:
"The user" reads awkwardly.  Please consider "You create a machine config..." or imperative, "Create a machine config..."

For reference, please see "Tone" > "Guidance for all tones" around p33 and "Pronouns" > "Gender-neutral pronoun" around p69 in the ISG.

bit of a nit, but I believe you can omit the mention of CRI-O.  Perhaps "Create a machine config manifest to partition the cluster management workloads."  Readers who want to know more, can read the [machine configuration tasks](https://docs.openshift.com/container-platform/4.7/post_installation_configuration/machine-configuration-tasks.html) in the post-installation configuration section.

another nit: instead of "specified" in the past tense, consider using present tense: "The cpuset that you specify must match the reserved cpuset that you specified in the Performance Addon Operator profile."

Aside: I'm not familiar with the material, so it might just be me, but is the first high-level step to configure a performance profile?  If so, consider making that the first high-level step.